ballerina/tests/call-procedures-test.bal

```diff
@@ -215,6 +214,8 @@ function testCallWithStringTypesOutParams() returns error? {
     test:assertEquals(paraCharactermax.get(string), "test2", "5th out parameter of procedure did not match.");
     test:assertEquals(paraCharacter.get(string), "b", "6th out parameter of procedure did not match.");
     test:assertEquals(paraNvarcharmax.get(string), "test3", "7th out parameter of procedure did not match.");
+
+    check ret.close();
```

Comment:
The test failed with "Failed to read OUT parameter value. No operations allowed after statement closed". The following changed fixed it. Please review.